CotEditor/Localizables/ServicesMenu.xcstrings

```diff
@@ -25,8 +25,8 @@
         },
         "es" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "Abrir fichero en CotEditor"
+            "state" : "translated",
+            "value" : "Nueva ventana de CotEditor que contiene la selección"
```

Comment:
I would like to follow Apple's TextEdit's similar wording for this phrase.
How about the expression “Ventana de CotEditor con selección”?
https://applelocalization.com/macos?q=Window+Containing+Selection&l=English&l=Spanish